<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>m_feed</class>
 <widget class="QDialog" name="m_feed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(112, 64, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#7f7f7f;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Home">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="botones.qrc">
      <normaloff>:/botonhome/imagenes/home.png</normaloff>:/botonhome/imagenes/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="New_post">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="botones.qrc">
      <normaloff>:/botonpost/imagenes/newpost.png</normaloff>:/botonpost/imagenes/newpost.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Friends">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="add_frend.qrc">
      <normaloff>:/addf/imagenes/add_f.svg</normaloff>:/addf/imagenes/add_f.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Log_out">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;
background-color:#ff545f;</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 96, 255);
color:#ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;  Nombre: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 96, 255);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="username">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ffffff;
color: #000000;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;  Usuario &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(112, 64, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="logo_icon.qrc">:/nuevo2/imagenes/user2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Amigos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="fotos.qrc">
      <normaloff>:/amigofoto/imagenes/amigos.png</normaloff>:/amigofoto/imagenes/amigos.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>100</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ffffff;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Publicaciones&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>741</width>
     <height>351</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff;
text-align: center;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_busqueda">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#000000;
color: #ffffff;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ordenar">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 0, 255);
color: #ffffff;
</string>
   </property>
   <property name="text">
    <string>Ordenar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filtrar_texto">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>550</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="filtrar_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#000000;
color: #ffffff;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filtrar_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>550</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 0, 255);
color: #ffffff;
</string>
   </property>
   <property name="text">
    <string>Ordenar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logo_icon.qrc"/>
  <include location="botones.qrc"/>
  <include location="add_frend.qrc"/>
  <include location="fotos.qrc"/>
 </resources>
 <connections/>
</ui>
